.gitignore

```diff
@@ -0,0 +1,4 @@
+# Eclipse
```

Comment:
This should be in y/our main git ignore file though.

Reply:
https://git-scm.com/docs/gitignore

Reply:
Hmmm interesting, yes I can set that up I guess. However, that doesn't really help in one-off clone situations.

Reply:
The global config applies to all repos.

Reply:
Sound like cases where Eclipse was not set up either.

Reply:
True, that's Eclipse's default import behavior

Reply:
It also doesn't apply inside Eclipse (for jgit ... there's likely another config for it ... but still).

Reply:
It does apply, I have global configuration and Eclipse/jgit does not care about them. Users should set their system accordingly to the IDEs they use (why just Eclipse?).

Reply:
Because that's what I was using. I can make it cover others if you like:

![image](https://github.com/user-attachments/assets/1f9c628f-dbb2-4e5e-b86d-b68eb6b940fd)
![image](https://github.com/user-attachments/assets/87945231-322b-4cac-83ed-59a7e680e998)

Reply:
Okay, I was just trying to make this convenient for people.
Do you want me to remove the .gitignore or add IDEA/NetBeans entries?

Reply:
I don't see the need for either but if you want.